scripting/get5/matchconfig.sp

```diff
@@ -104,19 +104,21 @@ stock bool LoadMatchConfig(const char[] config, bool restoreBackup = false) {
       }
     }
 
-    g_MapPoolList.GetString(Get5_GetMapNumber(), mapName, sizeof(mapName));
     ChangeState(Get5State_Warmup);
-
-    char currentMap[PLATFORM_MAX_PATH];
-    GetCurrentMap(currentMap, sizeof(currentMap));
-    if (!StrEqual(mapName, currentMap) && !restoreBackup) {
-      ChangeMap(mapName);
+    if (!restoreBackup) {
+      // When restoring from backup, changelevel is called after loading the match config.
+      // In all other cases, we want to load map 0 in the series.
+      g_MapPoolList.GetString(0, mapName, sizeof(mapName));
```

Comment:
We might not be able to put 0 here. If someone has `series_score` set in their match config, we shouldn't load map 0, should we? If it starts 1-0, we load map 1, skipping the first one?